### PR TITLE
Add migration to set the ExtractContent settings to true by default

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Add migration to set the ExtractContent settings to true by default

#### Ticket Link
[MM-35304](https://mattermost.atlassian.net/browse/MM-35304)

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by mgdelacroix — APPROVED** [review]

**Review by ethervoid — APPROVED** [review]
LGTM!

File: app/migrations.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ import (
 const EmojisPermissionsMigrationKey = "EmojisPermissionsMigrationComplete"
 const GuestRolesCreationMigrationKey = "GuestRolesCreationMigrationComplete"
 const SystemConsoleRolesCreationMigrationKey = "SystemConsoleRolesCreationMigrationComplete"
-const ContentExtractionConfigMigrationKey = "ContentExtractionConfigMigrationComplete"
+const ContentExtractionConfigDefaultTrueMigrationKey = "ContentExtractionConfigDefaultTrueMigrationComplete"
```
[the remark this answers is not in the document]
Comment (jespino) [reply app/migrations.go:19]: Yes and no, the old migration was removed but the constant was there, now is a new migration that has nothing to do with the old one, to ensure it is applied.

**Comment by esethna** [comment]
cc// @amyblais this is needed for v5.35

**Review by ogi-m — APPROVED** [review]
Tested and passed by Saturn. Approving.

**Comment by cpanato** [comment]
Cherry pick is scheduled.

**Pull request merged**